test/isEmpty.js

```diff
@@ -1,47 +1,28 @@
 var assert = require('assert');
 
-var R = require('..');
+var isEmpty = require('..').isEmpty;
```

Comment:
We used to do that for all our tests, and something drove us away from that.  I would rather keep the consistency, even if I can't recall the reason for it now.  I prefer it too, BTW, but find the consistency of the test suite more compelling.